web/src/components/molecules/Common/WorkspaceMenu/index.tsx

```diff
@@ -59,7 +79,7 @@ const WorkspaceMenu: React.FC<Props> = ({
 
   const onClick = (e: any) => {
```

Comment:
Please do a if/else if instead of just if/if. That way if the first if passes, it knows it doesn't need to check the next one.

Reply:
I left it like this for the future, as there will be more menu items.
I'll make it if/else and change it later to something more suitable